common/beerocks/bwl/nl80211/sta_wlan_hal_nl80211.h

```diff
@@ -12,6 +12,8 @@
 #include "base_wlan_hal_nl80211.h"
 #include <bwl/sta_wlan_hal.h>
 
+#define BUFFER_SIZE 4096
```

Comment:
To be consistent with nl82011 sta and mon HALs it's needed to define this macro in file `common/beerocks/bwl/nl80211/sta_wlan_hal_nl80211.cpp` and inside namespace `bwl::nl80211`

Reply:
I created a separate [issue](https://jira.prplfoundation.org/browse/PPM-232) for fixing commentaries

Reply:
I don't think this BUFFER_SIZE is even used?